docs/streaming-programming-guide.md

```diff
@@ -1937,6 +1937,16 @@ JavaPairDStream<String, String> unifiedStream = streamingContext.union(kafkaStre
 unifiedStream.print();
 {% endhighlight %}
 </div>
+<div data-lang="python" markdown="1">
+{% highlight python %}
+numStreams = 5
+kafkaStreams = []
+for x in range (0, numStreams):
+     kafkaStreams = x.map{ KafkaUtils.createStream(…)}
```

Comment:
Hm, I don't think this can be correct? x is an integer; you can't map it. Python collections don't map like that anyway. Are you just trying to `append` the new stream to `kafkaStreams`?

Nit: use `...` instead of the ellipsis character. 

Is the method really `show()` in Pyspark instead of `print()`?